bin/install_nvm_node_version

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+# This script expects a valid .nvmrc at the project's root
```

Comment:
If we expect this to be visible, we should test for that and fail if it's not there

Reply:
Command renamed to `nvm_install` and added a check for `.nvmrc` file in b71b2405e70c2e0b21e24e76d893bd9ba732dd71